packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -859,24 +873,34 @@ export const Table = <
             virtualItem={virtualItem}
             virtualizer={virtualizer}
           />
-          {subComponent && (
-            <WithCSSTransition in={row.isExpanded}>
```

Comment:
![image](https://github.com/user-attachments/assets/bcbeb373-91a7-439b-b360-c49c7d27dc7c)

Right, it doesn't work with `WithCSSTransition`, should I add a changeset saying that there will no longer be transition animation in sub-components?

Reply:
I'm not entirely sure but I have noticed one thing is the `in` props is being `undefined` as we pass in `row.isExpanded` which was true when the sub-component was still in the main row. However, since sub-component is now a sub-row, `row.isExpanded` is no longer useful as we return the sub-component instead. `Row` in sub-component does not represent the main that can expand and collapse anymore.

Reply:
Let's try to fix this in a separate PR.

Reply:
I have added this as an after-PR TODO.

Reply:
I have added the changeset.